ni/measurementlink/sessionmanagement/v1/session_management_service.proto

```diff
@@ -178,6 +234,34 @@ message ReserveSessionsRequest {
   // Optional. Timeout for the reservation request.
   // Allowed values: 0 (non-blocking, fails immediately if resources cannot be reserved), -1 (infinite timeout), or any other positive numeric value (wait for that number of milliseconds)
   int32 timeout_in_milliseconds = 4;
+
+  // Optional. Represents a set of annotations related to client machine details for session reservation or registration.
```

Comment:
I would update the documentation here and below to say:

"Optional. A set of annotations to add to the session for the duration of the reservation. These annotations will automatically be cleared when the session is unreserved."

Again, I would remove all of the documentation for well known keys.